Ontologies.Mappings/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json

```diff
@@ -139,17 +203,49 @@
       "OutputDtmi": "dtmi:com:willowinc:EnteringChilledWaterTemperatureSensor;1"
     },
     {
-      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Chilled_Water_Temperature_Setpoint;1",
+      "InputDtmi": "dtmi:mapped:core:Chilled_Water_Supply_Temperature_Setpoint;1",
       "OutputDtmi": "dtmi:com:willowinc:ChilledWaterTemperatureSetpoint;1"
     },
     {
-      "InputDtmi": "dtmi:org:w3id:rec:Company;1",
-      "OutputDtmi": "dtmi:com:willowinc:Company;1"
+      "InputDtmi": "dtmi:mapped:core:Chilled_Water_Usage_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:Sensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Class;1",
+      "OutputDtmi": "dtmi:com:willowinc:Class;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Close_Command;1",
+      "OutputDtmi": "dtmi:com:willowinc:ValveCloseActuator;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Close_Status;1",
+      "OutputDtmi": "dtmi:com:willowinc:PhaseBExportReactiveElectricalEnergySensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Cloud_Layer_1_Height_Code_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:CloudLayer1HeightCodeSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Cloud_Layer_2_Height_Code_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:CloudLayer2HeightCodeSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Cloud_Layer_3_Height_Code_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:CloudLayer3HeightCodeSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Coefficient_Of_Performance_Status;1",
+      "OutputDtmi": "dtmi:com:willowinc:CoefficientOfPerformance;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:Compressor_Status;1",
       "OutputDtmi": "dtmi:com:willowinc:State;1"
     },
+    {
+      "InputDtmi": "dtmi:mapped:core:Condenser_Water_Differential_Pressure_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:LeavingCondenserWaterPressureSensor;1"
```

Comment:
Is that a delta sensor? or should this be a CondenserWaterDeltaPressureSensor?

<img width="741" alt="image" src="https://github.com/mapped/mapped-willow-mapping/assets/84034746/3c55fa21-72d5-4951-b921-413e3fee65e1">